src/components/application_manager/test/resumption/resumption_data_db_test.cc

```diff
@@ -81,13 +85,17 @@ class ResumptionDataDBTest : public ResumptionDataTest {
 
   static void SetUpTestCase() {
     kDatabaseName = "resumption";
-    path_ = "test_storage";
-    CreateDirectory("./" + path_);
-    ON_CALL(mock_application_manager_settings_, app_storage_folder())
-        .WillByDefault(ReturnRef(path_));
-    test_db_ = new utils::dbms::SQLDatabase(kDatabaseName);
-    test_db_->set_path(path_ + "/");
-    res_db_ = new TestResumptionDataDB(mock_application_manager_settings_);
+    if (is_in_file) {
+      path_ = "test_storage";
+      CreateDirectory(file_system::CurrentWorkingDirectory() + "/" + path_);
+      CreateDirectory(kPath);
+      test_db_ = new utils::dbms::SQLDatabase(kDatabaseName);
+      test_db_->set_path(kPath + "/");
+      res_db_ = new TestResumptionDataDB(In_File_Storage);
+    } else {
+      res_db_ = new TestResumptionDataDB(In_Memory_Storage);
+      test_db_ = res_db_->get_db_handle();
+    }
```

Comment:
@okozlovlux I don't think that **switch** will be better in case when we have just boolean value